src/Hl7.Fhir.Base/Model/Base.cs

```diff
@@ -111,14 +113,21 @@ protected virtual bool TryGetValue(string key, [NotNullWhen(true)] out object? v
     // TODO bring Children + NamedChildren over as well.
 }
 
-
 /// <summary>
 /// A dynamic data type that can hold any element.
 /// </summary>
-public class DynamicDataType : DataType
+[Serializable]
+[DataContract]
+[FhirType("DynamicDataType","http://fire.ly/fhir/StructureDefinition/DynamicDataType")]
+public class DynamicDataType : DataType, IDynamicType
```

Comment:
Let's consider making another part of base, and keeping the Base.cs file as close to the core model as possible